sympy/parsing/c/c_parser.py

```diff
@@ -735,6 +736,72 @@ def transform_decl_stmt(self, node):
 
             return statement
 
+        def transform_paren_expr(self, node):
+            """Transformation function for Parenthesized expressions
+
+            Returns the result from its children nodes
+
+            """
+            return self.transform(next(node.get_children()))
+
+        def transform_unary_operator(self, node):
+            """Transformation function for handling unary operators
+
+            Returns
+            =======
+
+            unary_expression: Codegen AST node
+                    simplified unary expression represented as Codegen AST
+
+            Raises
+            ======
+
+            NotImplementedError if post increment/decrement operator,
```

Comment:
```suggestion
            NotImplementedError 
                If post increment/decrement operator,
```

Reply:
Thanks for the review, actually post increment/decrement operator are now supported, I forgot to remove that from the docstring